plugins/azure-devops/src/api/AzureDevOpsClient.ts

```diff
@@ -120,7 +120,23 @@ export class AzureDevOpsClient implements AzureDevOpsApi {
       queryString.append('repoName', repoName);
     }
     if (definitionName) {
-      queryString.append('definitionName', definitionName);
+      const definitionNames = definitionName.split(',');
```

Comment:
Seeing as `split()` will always return an array would the code not be easier to read by treating everything the same here? So we could just drop the `if (definitionNames.length > 1)` part and handle all cases in this loop. If they provide only 1 build definition it loops once, if they provide `n` build definitions then it loops `n` times